GoogleForCreators/web-stories-wp
# Redesign: Media Input

## Context
<img width="469" alt="Screenshot 2021-03-01 at 16 11 17" src="https://user-images.githubusercontent.com/3294597/109508690-6acb5a80-7aa0-11eb-9756-6bde195caef4.png">



## Summary
Adds Media input component to design-system


## Relevant Technical Choices



## To-do



## User-facing changes
N/A


## Testing Instructions
Open Storybook and verify the Media Input component looks like on Figma

Note: Dragging to add media is not in scope of this issue!


### QA






This PR can be tested by following these steps:

1.

### UAT







## Checklist




---



Fixes #6459
